src/packing_queue/PackingQueue.jsx

```diff
@@ -93,12 +116,10 @@ const PackingQueue = () => {
         pageNumber
       )
         .then((data) => {
-          if (data) {
-            if (isMounted) {
-              let tableData = extractHistoryDetails(data?.packingSlips);
-              setFilteredHist(tableData);
-              setHistTotalCount(data?.totalCount);
-            }
+          if (data && isMounted) {
+            let tableData = extractHistoryDetails(data?.packingSlips);
```

Comment:
Does this need to be a let? can it be a const or just directly place it in the setFiltertedHist